packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -133,7 +133,9 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
         fontSize: 'inherit',
         color: palette.neutralPrimary,
         backgroundColor: 'transparent',
-        flex: '1 1 0',
+        flexBasis: 0,
```

Comment:
why change this? it's more bytes-over-wire

Reply:
I changed it because IE didn't support this shorthand format:
`flex: '1 1 0'`

`flex: '1 1 0px'` does work, however. I've updated the branch.

BTW: This fixes an issue where the clear button is incorrectly positioned:

![image](https://user-images.githubusercontent.com/6515360/35083789-27530ace-fc21-11e7-8390-a2152b66d999.png)

Reply:
Ah, good to know we need the px on the flex-basis part no matter what, thanks for catching that.